src/plugins/intel_cpu/src/emitters/jit_eltwise_emitters.hpp

```diff
@@ -611,5 +610,90 @@ class jit_soft_sign_emitter : public jit_emitter {
     void register_table_entries() override;
 };
 
+class jit_is_finite_emitter : public jit_emitter {
+public:
+    jit_is_finite_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t hostIsa,
+                    InferenceEngine::Precision execPrc = InferenceEngine::Precision::FP32) : jit_emitter(host, hostIsa, execPrc) {
+        prepare_table();
+    }
+    jit_is_finite_emitter(dnnl::impl::cpu::x64::jit_generator *host, dnnl::impl::cpu::x64::cpu_isa_t hostIsa, const std::shared_ptr<ov::Node>& node,
+                    InferenceEngine::Precision execPrc = InferenceEngine::Precision::FP32) : jit_emitter(host, hostIsa, node, execPrc) {
+        prepare_table();
+    }
+
+    size_t get_inputs_num() const override { return 1; };
+    size_t aux_vecs_count() const override { return host_isa_ == dnnl::impl::cpu::x64::avx512_core ? 0 : 2; }
```

Comment:
Why we need 2 vecs for non-avx512 impl? Looks like non of them are used

Reply:
Removed